xCertificateServices/DSCResources/MSFT_xAdcsOnlineResponder/MSFT_xAdcsOnlineResponder.schema.mof

```diff
@@ -0,0 +1,7 @@
+[ClassVersion("0.1.0.0"), FriendlyName("xAdcsOnlineResponder")]
+class MSFT_xAdcsOnlineResponder : OMI_BaseResource
+{
+    [Required, Description("If the Online Responder service is configured to use Standalone certification authority, then an account that is a member of the local Administrators on the CA is required. If the Online Responder service is configured to use an Enterprise CA, then an account that is a member of Domain Admins is required."), EmbeddedInstance("MSFT_Credential")] String Credential;
+    [Required, Description("Specifies whether the Online Responder feature should be installed or uninstalled."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Key] String Name;
```

Comment:
Excellent! I knew there must have been a proper way to do this :smile:

I'll make the changes to xADCSOnlineResponder. Is it worth me submitting a separate PR to change this in the xADCSWebEnrollment resource as well? It would seem best to have these resources work in the same way.